aip/cloud/25146.md

```diff
@@ -0,0 +1,28 @@
+---
+id: 25146
+state: draft
+created: TBD
+placement:
+  category: gcp
+---
+
+# Generic Fields
+
+Guidelines related to generic fields in [AIP-146](../0146.md) **should** be applied
+with the following exceptions.
+
+## Maps
+
+Maps with nested object types **must** not exist.
+
+The rationale is:
+- Declarative tools cannot represent schemas for maps with complex value types.
+
+### Alternative
+
+APIs considering a map should prefer lists with UNORDERED_LIST behavior where the
```

Comment:
Use backticks for "UNORDERED_LIST" and maybe we should link to the field_behavior AIP so that people know what you mean by "behavior"?